src/components/LeafletMap.js

```diff
@@ -120,6 +120,7 @@ export class LeafletMap {
 
   handleAddLayerError = (error) => {
     console.error(error);
+    this.toggleLoadingIndicator(true); //show loading indicator
```

Comment:
@timhitchins wouldn't we want to do the opposite here? If a layer fails to fetch we don't want the user to be stuck with a modal overlay that's blocking them from interacting with the map.

Reply:
Good point.